static/app/views/issueDetails/groupRelatedIssues/index.tsx

```diff
@@ -0,0 +1,92 @@
+// XXX: A lot of the UI for this file will be changed once we use IssueListActions
+// We're using GroupList to help us iterate quickly
+import {Fragment} from 'react';
+import type {RouteComponentProps} from 'react-router';
+import styled from '@emotion/styled';
+
+import Feature from 'sentry/components/acl/feature';
+import GroupList from 'sentry/components/issues/groupList';
+import * as Layout from 'sentry/components/layouts/thirds';
+import LoadingError from 'sentry/components/loadingError';
+import LoadingIndicator from 'sentry/components/loadingIndicator';
+import {t} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+import {useApiQuery} from 'sentry/utils/queryClient';
+
+type RouteParams = {
+  groupId: string;
+  orgId: string; // XXX: Find caller and rename from orgId to orgSlug
+};
+
+type Props = RouteComponentProps<RouteParams, {}>;
+
+type RelatedIssuesResponse = {
+  same_root_cause: number[];
```

Comment:
Can we camel-case the response to `sameRootCause`?

Reply:
That's not the casing for Python projects.
I may be able to come up with a key that is a single word and satisfy both worlds.

Do you want me to pursue it?

Reply:
It's not that big of a deal to me, but other endpoints seem to camel case response data in the serializers ([example](https://github.com/getsentry/sentry/blob/master/src/sentry/api/serializers/models/group.py#L320-L350)) so it seems like something we could do for consistency. Not sure if we actually enforce that at all though?

Reply:
I will be fixing it in my next BE PR.